modules/gitops-argo-cd-installation.adoc

```diff
@@ -17,10 +17,10 @@ To manage cluster configurations or deploy applications, you can install and dep
 
 . Select *Openshift GitOps Operator* from the installed operators and select the *Argo CD* tab.
 
-. Click *Create* to configure the following parameters:
+. Click *Create* to configure the parameters:
 
-.. For *Name*, enter `/`.
+.. By default the *Name* is set as *argocd*. Enter the *Name* of the instance.
```

Comment:
nit: Enter the **Name** of the instance. By default the **Name** is set as **argocd**.